belgif-rest-problem/src/main/java/io/github/belgif/rest/problem/api/InputValidationIssues.java

```diff
@@ -72,6 +74,28 @@ public static InputValidationIssue invalidStructure(InEnum in, String name, Obje
                         .in(in, name, value);
     }
 
+    public static <T extends Comparable<T>> InputValidationIssue outOfRange(InEnum in, String name, T value, T min,
+            T max) {
+        if (min == null && max == null) {
+            throw new IllegalArgumentException("At least one of min, max must be non-null");
+        }
+        InputValidationIssue issue =
+                new InputValidationIssue(ISSUE_TYPE_INVALID_STRUCTURE, "Input value is out of range")
```

Comment:
Shouldn't this be `ISSUE_TYPE_OUT_OF_RANGE`?

Reply:
Yes :(

It demonstrates we need dedicated tests for the InputValidationIssues class, because it is used in other tests that assume it works correctly. I will do that in another general PR that improves the tests.